utils/query.ts

```diff
@@ -0,0 +1,62 @@
+import { Client } from "tina-graphql-gateway";
```

Comment:
Yeah I see what you mean, though the query itself is squarely a "userland" concern, so for all we know and care it could be put wherever they want it, whereas for the `schema.ts` we need to know where it is. Part of the challenge with the cloud starter is getting the point across without too much ceremony, and then getting out of the way. So I want to be careful not to over-abstract things out of the file where it's being used. Definitely a tricky balance, so lets chat about it some more.